tests/operations/index.mjs

```diff
@@ -83,6 +83,7 @@ import "./tests/Media";
 import "./tests/ToFromInsensitiveRegex";
 import "./tests/YARA.mjs";
 import "./tests/ConvertCoordinateFormat";
+import "./tests/operations/ByteStringLiteral";
```

Comment:
Should be `import "./tests/ByteStringLiteral";` I think

Reply:
Thank you very much for your support. I gave up. I will try to follow your guidelines this week-end.

Reply:
done